docs/manual/java/guide/cluster/Cluster.md

```diff
@@ -42,23 +42,71 @@ In development you are typically only running the service on one cluster node. N
 
 ### Joining during production
 
-Starting from version 1.5.0, Lagom offers support for [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/current/bootstrap/). Akka Cluster Bootstrap is enabled by default in production mode and disabled in development and test mode.
+Starting from version 1.5.0, Lagom offers support for [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/1.0.0-RC2/bootstrap/). Akka Cluster Bootstrap is enabled by default in production mode and disabled in development and test mode.
```

Comment:
I think the first sentence should make it very obvious that starting `1.5.0` cluster bootstrap will try to take over unless seed-nodes exist. It is important to specify that cluster bootstrap will even take precedence over `join-self` specially considering we have users that rely on `join-self`.

I think a small paragraph stating the rules:

 * `if` the user defined `seed-nodes` use them
 * `else if` cluster bootstrap is enabled use bootstrap
      * `else if` `join-self` is enabled join self